index.html

```diff
@@ -24,6 +24,11 @@
         email: "hta@google.com",
         company: "Google Inc.",
       },
+      {
+        name: "Varun Singh",
+        email: "varun@varunsingh.net",
+        company: "Expert",
```

Comment:
@dontcallmedom can "Expert" be put here, or should one omit company in this case?

Reply:
removed, or "Invited Expert" would work